### PR TITLE
Replacing APM svg images with jsx files and replacing klarna image with sofort

### PR DESCRIPTION
**Pull request by aditya27marri**
replacing all the APM svgs with the jsx files
klarna image was against the legal requirement so we replaced it with the new sofort image

### CODE REVIEW

File: src/resources/fundingLogos/mybank.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+/* @flow */
+/* @jsx jsxToHTML */
+
+import { jsxToHTML, JsxHTMLNode } from '../../lib/jsx'; // eslint-disable-line no-unused-vars
+import { BUTTON_LOGO_COLOR } from '../../constants';
+
+const MYBANK_LOGO_COLORS = {
+    [ BUTTON_LOGO_COLOR.WHITE ]: {
+        primary:   '#ffffff',
+        secondary: '#ffffff',
```
Comment (mcntsh) [inline src/resources/fundingLogos/mybank.jsx]: Won't having all paths be the same color create a blank-looking SVG? I think that's what's happening here in [this screenshot](https://github.com/paypal/paypal-checkout/blob/d70d9d19ee3e7dfbb9c288e2e43ad939caf2adb0/test/screenshot/images/button.locale%3Dde_DE%26button.commit%3Dtrue%26button.style.color%3Dblack%26button.style.layout%3Dvertical%26button.style.maxbuttons%3D4%26button.funding.allowed%3Dcredit%2Celv%2Cgiropay%2Csofort.png)

File: src/resources/fundingLogos/sofort.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+/* @flow */
+/* @jsx jsxToHTML */
+
+import { jsxToHTML, JsxHTMLNode } from '../../lib/jsx'; // eslint-disable-line no-unused-vars
+import { BUTTON_LOGO_COLOR } from '../../constants';
+
+const SOFORT_LOGO_COLORS = {
+    [ BUTTON_LOGO_COLOR.WHITE ]: {
+        primary:   '#ffffff',
```
Comment (mcntsh) [inline src/resources/fundingLogos/sofort.jsx]: Similar comment as MyBank; these two matching path colors is going to [white-out the logo](https://github.com/paypal/paypal-checkout/blob/d70d9d19ee3e7dfbb9c288e2e43ad939caf2adb0/test/screenshot/images/button.locale%3Dnl_NL%26button.commit%3Dtrue%26button.style.color%3Dblack%26button.style.layout%3Dvertical%26button.style.maxbuttons%3D4%26button.funding.allowed%3Dcredit%2Cideal%2Csofort%2Ccard.png).

Comment (mcntsh) [reply src/resources/fundingLogos/sofort.jsx]: Also, the Sofort logo looks a bit funky [having a containing box](https://github.com/paypal/paypal-checkout/blob/d70d9d19ee3e7dfbb9c288e2e43ad939caf2adb0/test/screenshot/images/button.locale%3Dnl_NL%26button.commit%3Dtrue%26button.style.color%3Dsilver%26button.style.layout%3Dvertical%26button.style.maxbuttons%3D4%26button.funding.allowed%3Dcredit%2Cideal%2Csofort%2Ccard.png). Is it possible to use a version of this logo without the box?

Comment (aditya27marri) [reply src/resources/fundingLogos/sofort.jsx]: we cannot take out the pink background as that what the legal team wants. as it is their main color

File: src/resources/fundingLogos/mybank.jsx (hunk 1 above)
Comment (aditya27marri) [reply src/resources/fundingLogos/mybank.jsx]: Fixed this . please check it now

Comment (aditya27marri) [reply src/resources/fundingLogos/mybank.jsx]: Fixed it now. please check it again

**Comment by mcntsh** [comment]
@aditya27marri Tests look good. Could you fetch rebase against master and resolve the conflicts and then we should be good to go ✌️

**Comment by aditya27marri** [comment]
I fixed them all. Please check it now

**Comment by mcntsh** [comment]
LGTM. Wish Klarna had a better logo solution for us, but it's what we have. Thanks consolidating those other APM logos into `jsx` files.

**Pull request merged**